app/src/main/java/org/fossasia/openevent/app/common/Constants.java

```diff
@@ -21,6 +21,10 @@
     public static final String PREF_PAYMENT_ACCEPT_ONSITE = "accept_onsite";
     public static final String PREF_PAYMENT_ONSITE_DETAILS = "onsite_details";
     public static final String PREF_PAYMENT_COUNTRY = "key";
+    public static final String PREF_SCAN_WILL_CHECK_IN = "check_in";
+    public static final String PREF_SCAN_WILL_CHECK_OUT = "check_out";
+    public static final String PREF_SCAN_WILL_VALIDATE = "validate";
```

Comment:
Add these locally where they are used

Reply:
These are getting used in both ScanSettings and ScanQRPresenter, like other constants. Need to move them to ScanSettings?

Reply:
Let it be